Steps incorrect.

cunno89 opened the pull request:
EXTRACT is not a valid command. I've seen folks alias it for extrac32, but as this doc is written it will not work. You can ping me at mrajess@microsoft.com if you have any questions.

P.S. Just realized I messed up. I didn't mean to edit as much as I did. Just meant to rewrite

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (luisca). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

cunno89 committed:
Steps incorrect.

EXTRACT is not a valid command on any version of windows. The steps as given do not work. Also removed mention of IExpress as it was random and unrelated and the link was dead.
